src/main/java/com/metsci/laproc/datareference/InputDataReferenceImpl.java

```diff
@@ -10,14 +10,13 @@
 
 import java.util.ArrayList;
 import java.util.HashMap;
-import java.util.Iterator;
 import java.util.List;
 
 /**
  * This reference allows tools to access the graph and associated data
```

Comment:
"This reference allows tools to access the raw input data represented as ClassifierDataSets"